src/components/Button/index.module.scss

```diff
@@ -17,11 +17,23 @@ $curve: cubic-bezier(.165, .84, .44, 1);
 
```

Comment:
@paulpopus Button changes look fine - did you verify that nothing needed to be changed with the previous blocks that have been built? e.g. `card-grid`, your `sticky-list` block, `FeaturedMediaGallery` block?

Not extremely crucial but if its not taken care of now, then its just another thing we need to add to the list once review process starts